### ISSUE TITLE
Linux VST3 binary release should be in a folder

### ISSUE THREAD

**Issue by crshrprt** [attilammagyar/js80p#26]
Thank you for this interesting plugin.

I noticed that the Linux binary release zip file contains among other files the plugin binary called either js80p.vst3 or js80p.so.

I can't use the file as it is in the binary release zip file. 

To make it work with the plugin hosts/DAWs I use, mainly Bespoke Synth,  Ardour and Carla it needs to be put inside a correct VST3 bundle folder structure ( [Steinberg VST3 Plug-in Format Structure docs](https://steinbergmedia.github.io/vst3_dev_portal/pages/Technical+Documentation/Locations+Format/Plugin+Format.html#for-the-linux-platform) ).

In short, the zip files should contain a folder and files with this structure:
```
js80p.vst3/
└── Contents
    └── x86_64-linux
        └── js80p.so
```

EDIT: I see you already have a merged release with the correct structure, so this problem should be addressed only for the Linux VST3 single file?

**Comment by attilammagyar** [comment]
The single file VST3 distributions exists because [some hosts on some platforms have trouble with loading VST3 bundles](https://github.com/steinbergmedia/vst3sdk/issues/115).

But since issue #20 and JS80P v1.8.0, VST3 bundles are available for both [AVX compatible x64 systems](https://github.com/attilammagyar/js80p/releases/download/v2.3.2/js80p-2_3_2-avx-vst3_bundle.zip) and [SSE2 compatible x86 and x64 systems](https://github.com/attilammagyar/js80p/releases/download/v2.3.2/js80p-2_3_2-sse2-vst3_bundle.zip). Inside these ZIP files you can find the `js80p.vst3` bundle directory that you can copy to `~/.vst3` on Linux and to `C:\Users\YourUserName\AppData\Local\Programs\Common\VST3` on Windows.

**Comment by attilammagyar** [comment]
> EDIT: I see you already have a merged release with the correct structure, so this problem should be addressed only for the Linux VST3 single file?

No, because the single file distributions with the deprecated format are intended to be used when a host fails to recognize the plugin from the bundle due to the `uname()` kerfuffle. If they were using the same directory structure, then they would fail to be loaded the same way. If a host suffers from the `uname()` problem and doesn't support the legacy format at the same time, then we're out of luck.

Though you're right in that [the official SDK code doesn't seem to support the single SO file named as `.vst3` format on Linux](https://github.com/steinbergmedia/vst3_public_sdk/blob/e61ee76eb0f8771a05e99018bc4705ac1b603e48/source/vst/hosting/module_linux.cpp#L146), there are hosts that can load the plugin from this format despite the official SDK; one example is Reaper, which also happens to suffer from the `uname()` problem.
